GsaGH/Components/6_Display/ReactionForceDiagrams.cs

```diff
@@ -143,11 +143,6 @@ protected override void AppendAdditionalComponentMenuItems(ToolStripDropDown men
         momentUnitsMenu,
       };
 
-      if (_lengthUnit == LengthUnit.Undefined) {
-        ToolStripMenuItem modelUnitsMenu = GenerateModelGeometryUnitsMenu("Model geometry");
-        toolStripItems.Insert(0, modelUnitsMenu);
-      }
-
```

Comment:
why did we remove those?

Reply:
undefined no longer exist. So, Have set default unit to rhinolength unit if active doc not null else meter.  https://github.com/arup-group/GSA-Grasshopper/pull/696/files#diff-6b17d5f6ba38a8a578ae64ec954c7450c81f98364f66ee7229268e5cf7efd5c5